src/middlewares/authorization.js

```diff
@@ -0,0 +1,15 @@
+import verifyToken from '../utils/authentication';
+
+const checkToken = (req, res, next) => {
+  const token = req.headers['x-access-token'] || req.query.token || req.headers.authorization;
+  if (token) {
+    verifyToken(token);
+    return next();
```

Comment:
you need to ensure the token is valid before calling the next method. The `verifyToken` method should return a boolean `success` property that determines if the token is valid or not.

Reply:
I saw the way you implemented the verifyToken below. Use the returned value fro the method appropriately.